WeakAuras/Prototypes.lua

```diff
@@ -4839,49 +4839,60 @@ WeakAuras.event_prototypes = {
           local count = 0;
           local names = "";
           local units = {};
+          local activation = %s;
+          activation = activation and WeakAuras.IsTriggerActive(aura_env.id) or not activation;
```

Comment:
That doesn't work at all. Triggers can't depend on other triggers.

Reply:
Mhmm i'm not surprised of reaction.
I still really don't like to run the "group" and "multi" rangecheck for a full encounter.

What could be done is to leave it like that without my last dirty commit, but add a performance warning, and add rangechecks to conditions for triggers where it can be relevant, like aura, bw and dbm timers, and cast trigger.

Do you think this is an acceptable solution ?